contracts/test/contract.SL2RD.test.js

```diff
@@ -332,4 +464,46 @@ contract("SL2RD", (accounts) => {
         });
     }
   });
+
+  specify(
+    "Transfer next available slot to a specified address",
+    async () => {
+      const shareContract = await SHARE.deployed();
+      const splitContract = await SL2RD.new();
+      const operatorRegistry = await OperatorRegistry.deployed();
+      const ownerAddresses = Array(10).fill(accounts[0]);
+      const recipientAddress = accounts[NON_OWNER_ADDRESS_INDEX];
+      const uniformCollaboratorsIds = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+      const communitySplitsPercentage = 4000;
+
+      await splitContract.initialize(
+        ownerAddresses /* addresses_ */,
+        uniformCollaboratorsIds /* tokenIds_ */,
+        shareContract.address /* shareContractAddress_ */,
+        communitySplitsPercentage /* communitySplitsPercentage_ */,
+        operatorRegistry.address /* operatorRegistryAddress_ */
+      );
+
+      // Transfer the reserved community allocation (first 4 slots) to community
+      for (let i = 0; i < 4; i++) {
+        await splitContract.transferNextAvailable(recipientAddress, {
```

Comment:
Which address is the caller of this function?

Reply:
the owner of all the tokenIds initially.